types/cached/index.d.ts

```diff
@@ -1,64 +1,63 @@
-// Type definitions for cached 5.0
+// Type definitions for cached 6.0
 // Project: https://github.com/groupon/node-cached
 // Definitions by: Juraj Mäsiar <https://github.com/Juraj-Masiar>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 // TypeScript Version: 4.0
 
-import Bluebird = require("bluebird");
 import Memcached = require('memcached');
 
 export = cached;
 
 interface CacheOptions {
-    defaults?: CacheDefaults | undefined;
+    defaults?: CacheDefaults;
     backend: BackendOptions;
 }
 
 interface NewCacheOptions extends CacheOptions {
-    name?: string | undefined;
+    name?: string;
 }
 
 interface CacheDefaults {
     /**
      * expire is the time in seconds after which a value should be deleted from the cache (or whatever expiring natively means for the backend).
      * Usually you'd want this to be 0 (never expire).
      */
-    expire?: number | undefined;
+    expire?: number;
     /**
      * freshFor is the time in seconds after which a value should be replaced.
      * Replacing the value is done in the background and while the new value is generated (e.g. data is fetched from some service) the stale value is returned.
      * Think of freshFor as a smarter expire.
      */
-    freshFor?: number | undefined;
+    freshFor?: number;
     /**
      * timeout is the maximum time in milliseconds to wait for cache operations to complete.
      * Configuring a timeout ensures that all get, set, and unset operations fail fast.
      * Otherwise there will be situations where one of the cache hosts goes down and reads hang for minutes while the memcached client retries to establish a connection.
      * It's highly recommended to set a timeout. If timeout is left undefined, no timeout will be set and the operations will only fail once the underlying client, e.g. memcached, gave up.
      */
-    timeout?: number | undefined;
+    timeout?: number;
 }
 
 interface BackendOptions {
     type: 'memcached' | 'memory' | 'noop';
-    client?: Memcached | undefined;
-    hosts?: string | undefined;
-    poolSize?: number | undefined;
+    client?: Memcached;
```

Comment:
Just to clarify, these are for https://www.typescriptlang.org/docs/handbook/release-notes/typescript-4-4.html#exact-optional-property-types---exactoptionalpropertytypes - are you sure you want to remove the distinction?

Reply:
Yes, I've removed those `undefined` types because of the new warning I've got when running the test here.
But from the caller point of view it should be the same, right?
Anyway, this is a major version change so API changes are expected :)

Reply:
Cool, seems reasonable then 👍🏻